common/appointment/create.ts

```diff
@@ -35,48 +36,64 @@ export abstract class AppointmentCreateGroupInput extends AppointmentCreateInput
     appointmentType: 'group';
 }
 
+export const isAppointmentOneWeekLater = (appointmentDate: Date) => {
+    const now = moment.now();
+    const start = moment(appointmentDate);
+    const diffDays = start.diff(now, 'days');
+    return diffDays >= 6 ? true : false;
```

Comment:
Shouldn't it be greater than six?

```suggestion
    return diffDays > 6;
```